src/video/video.js

```diff
@@ -340,23 +340,34 @@
                 var parent = me.video.renderer.getScreenCanvas().parentNode;
                 var _max_width = Math.min(maxWidth, parent.width || window.innerWidth);
                 var _max_height = Math.min(maxHeight, parent.height || window.innerHeight);
-
-                if (settings.maintainAspectRatio) {
-                    // make sure we maintain the original aspect ratio
-                    var designRatio = me.video.renderer.getWidth() / me.video.renderer.getHeight();
-                    var screenRatio = _max_width / _max_height;
+                var designRatio = me.video.renderer.getWidth() / me.video.renderer.getHeight();
+                var screenRatio = _max_width / _max_height;
+                var sWidth = me.video.renderer.getHeight() * screenRatio;
+                var sHeight = me.video.renderer.getWidth() * (window.innerHeight / window.innerWidth);
```

Comment:
yes I would to be honest, especially since you already use `sRatio` that already takes them in account : 
````javascript
var screenRatio = _max_width / _max_height;
````

then for me, good to merge as well ;)